### PR TITLE
Remove "ADB over network" preference if DISALLOW_DEBUGGING_FEATURES is set

### PR DESCRIPTION
**Pull request by abioteau**
Fix issue https://github.com/sonyxperiadev/bug_tracker/issues/133

### CODE REVIEW

**Comment by jerpelea** [comment]
@luk1337 @Myself5

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java [hunk 1]
```diff
@@ -244,13 +246,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
...
         findPreference(mDispCalSwitchPref).setOnPreferenceChangeListener(mPreferenceListener);
 
-        String adbN = SystemProperties.get(PREF_ADB_NETWORK_READ);
-        boolean adbNB = isNumeric(adbN) && (Integer.parseInt(adbN) > 0);
-        mPrefEditor.putBoolean(mADBOverNetworkSwitchPref, adbNB);
-        SwitchPreference adbon = (SwitchPreference) findPreference(mADBOverNetworkSwitchPref);
-        // Set the switch state accordingly to the Preference
-        adbon.setChecked(adbNB);
-        updateADBSummary(adbNB);
+        mUserManager = (UserManager) getSystemService(Context.USER_SERVICE);
```
Comment (luca020400) [inline src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java]: You can avoid the cast and use
getSystemService(UserManager.class)

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java [hunk 2]
```diff
@@ -244,13 +246,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
...
+        if (mUserManager.hasUserRestriction(UserManager.DISALLOW_DEBUGGING_FEATURES)) {
+            SystemProperties.set(PREF_ADB_NETWORK_COM, "-1");
+            getPreferenceScreen().removePreference(findPreference(mADBOverNetworkSwitchPref));
+        } else {
+            String adbN = SystemProperties.get(PREF_ADB_NETWORK_READ);
```
Comment (luca020400) [inline src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java]: boolean adbNB = SystemProperties.getInt(PREF_ADB_NETWORK_READ, 0) > 0;

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java (hunk 1 above)
Comment (abioteau) [reply src/sonyxperiadev/extendedsettings/ExtendedSettingsFragment.java]: Done

**Comment by abioteau** [comment]
@jerpelea there is another commit for last comment of @luca020400 

https://github.com/abioteau/packages_apps_ExtendedSettings/commit/e1ef3de89b8ea2bb79f7b71fc37ff7491df4b082

**Comment by jerpelea** [comment]
push it

**Pull request merged**